src/wp-includes/class-wp-script-modules.php

```diff
@@ -225,6 +226,35 @@ public function print_import_map() {
 		}
 	}
 
+	/**
+	 * Prints the necessary script to load import map polyfill for browsers that
+	 * do not support import maps.
+	 *
+	 * @since 6.5.0
+	 */
+	public function print_import_map_polyfill() {
+		$import_map = $this->get_import_map();
+		if ( ! empty( $import_map['imports'] ) ) {
+			$test = 'HTMLScriptElement.supports && HTMLScriptElement.supports("importmap")';
+			$src  = includes_url( 'js/dist/vendor/wp-polyfill-importmap.min.js' );
+
+			echo (
```

Comment:
Have you tried using `wp_get_script_polyfill`?

Reply:
Done in https://github.com/WordPress/wordpress-develop/pull/5947/commits/4b23ba81ddb067110e41d05550de7f2a4f09dad3